.github/workflows/ondemand-main.yaml

```
@@ -8,33 +8,34 @@ on:
         default: 'maintenance-3.0.x'
         required: true
       sauce_labs:
-        description: 'Should unit tests be run on Sauce Labs?'
+        description: 'Run Unit tests on Sauce Labs?'
```

For consistency with the descriptions below:
```suggestion
        description: 'Run unit tests on Sauce Labs?'
```